lib/block_view.go

```diff
@@ -3082,15 +3090,40 @@ func (bav *UtxoView) _connectUpdateGlobalParams(
 			}
 		}
 		if len(extraData[FeeBucketRateMultiplierBasisPointsKey]) > 0 {
-			newGlobalParamsEntry.FeeBucketRateMultiplierBasisPoints, bytesRead = Uvarint(
+			val, bytesRead := Uvarint(
 				extraData[FeeBucketRateMultiplierBasisPointsKey],
 			)
+			if val > _basisPoints {
```

Comment:
Not sure why I originally named this const `_basisPoints`... The appropriate name is `_maxBasisPoints`. Do you want to rename it in a followup PR? I can do it otherwise.